core/src/main/java/c/org/rajawali3d/object/renderers/ObjectRendererImpl.java

```diff
@@ -171,9 +171,14 @@ private void applyDepth() {
     }
 
     private void checkDoubleSided(@NonNull ObjectRenderer lastUsed) {
-        if (isDoubleSided && !lastUsed.isDoubleSided()) {
-            // We are double sided, the last renderer is not - disable culling
-            GLES20.glDisable(GLES20.GL_CULL_FACE);
+        if (isDoubleSided) {
```

Comment:
Why was this expanded into more verbose logic?

Reply:
If isDoubleSided is true, but the lastUsed was also, then the else clause would be used...

Reply:
But in that situation, there is nothing to do regarding double sided, which is why the clause was omitted originally.

Reply:
The original else clause assumed _not_ double sided and changed GL, so if the condition fails because both the current and last used are doubleSided, then GL will be configured for non-double-sided...

Reply:
Should have said "The top-level else clause assumes _not_ double sided and changesGL..."